src/main/java/net/pms/configuration/FormatConfiguration.java

```diff
@@ -96,17 +100,24 @@
 	public static final String MPEGPS = "mpegps";
 	public static final String MPEGTS = "mpegts";
 	public static final String OGG = "ogg";
+	/** OGG container with only audio track */
+	public static final String OGA = "oga";
 	public static final String OPUS = "opus";
 	public static final String PCX = "pcx";
 	public static final String PNG = "png";
 	public static final String PNM = "pnm";
 	public static final String PSD = "psd";
 	public static final String QDESIGN = "qdmc";
+	/** This is the RealAudio file format, not one of the codecs */
 	public static final String RA = "ra";
 	public static final String RAW = "raw";
-	public static final String REALAUDIO_LOSSLESS = "ralf";
+	public static final String REALAUDIO_14_4 = "ra14.4";
+	public static final String REALAUDIO_28_8 = "ra28.8";
+	public static final String RALF = "ralf";
```

Comment:
The order should be readable if set after RA. It's just minor formatting though.